scripts/quests/sandoria/Distant_Loyalties.lua

```diff
@@ -0,0 +1,170 @@
+-----------------------------------
+-- Distant Loyalties
+-----------------------------------
+-- !addquest 0 74
+-- Southern San 'dOria !zone 230
+-- Femitte: !pos -17.7129 2.1000 10.1636
+-- Bastok Markets !zone 235
+-- Michea: !pos -299.2114 -15.9717 -156.0629
+-- Mythril Ingot !giveitem 653
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.SANDORIA, xi.quest.id.sandoria.DISTANT_LOYALTIES)
+
+quest.reward =
+{
+    fame     = 30,
+    fameArea = xi.fameArea.SANDORIA,
+    item     = xi.item.WHITE_CAPE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:getFameLevel(xi.fameArea.SANDORIA) >= 4
+        end,
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Femitte'] = 
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(663)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [663] = function(player, csid, option, npc)
+                    if option == 0 then
+                        quest:begin(player)
+                        npcUtil.giveKeyItem(player, xi.ki.GOLDSMITHING_ORDER)
+                        quest:setVar(player, 'Prog', 1)
```

Comment:
You're switching states here from available to accepted, no need to set a Prog var